resteasy-core/src/main/java/org/jboss/resteasy/tracing/RESTEasyTracingLoggerImpl.java

```diff
@@ -134,7 +137,16 @@ private void logImpl(final RESTEasyTracingEvent event, final long duration, fina
          }
          if (logger.isEnabled(loggingLevel)) {
             logger.log(loggingLevel,
-                     event.name() + ' ' + message.toString() + " [" + tracingInfo.formatDuration(duration) + " ms]");
+                    new StringBuilder()
+                            .append(requestId)
+                            .append(' ')
+                            .append(event.name())
+                            .append(' ')
+                            .append(message.toString())
+                            .append(" [")
+                            .append(tracingInfo.formatDuration(duration))
+                            .append(" ms]")
+                            .toString());
```

Comment:
This is fine, however, just FYI, it was not needed in this case as the JIT compiler would have optimized the code exactly the way you did in this commit.

Reply:
@asoldano okay! should I change revert it back then?

Reply:
it looks very verbose, let me revert it :-)

Reply:
Nevermind, I've merged the PR as I already pushed a snapshot of the resteasy-extensions and hence this was needed to solve the master build being broken by that ;-)